docs: safeguard improvements (#952)
elastisys/compliantkubernetes: docs/user-guide/safeguards/enforce-minimum-replicas.md

# Avoid Downtime with Replicas

!!!important

>     - This safeguard is enabled by default with the enforcement action `warn`. As a result, resources that violate this policy will generate warning messages, but will still be created.

Author: I'm considering updating all of these texts to something like this. I was feeling that "enforcement action: warn/deny" seemed to be a odd wording for someone who has not looked at our configuration. What do you think of this?
```suggestion
    - This safeguard is enabled by default and will warn on violations. As a result, resources that violate this policy will generate warning messages, but will still be created.
```

Reviewer: I agree and think the change is much better. More "regular" english that actually explains what it means :+1:

Author: Updated now
